### PR TITLE
feat: add custom authselect profile

### PR DESCRIPTION
**Pull request by wandapeter**
profile tested locally. can you create a staging tag?

#310

### CODE REVIEW

**Comment by RoyalOughtness** [comment]
here you go https://github.com/secureblue/secureblue/tree/staging

File: files/system/usr/etc/authselect/custom/secureblue/REQUIREMENTS [hunk 1]
```diff
@@ -0,0 +1 @@
+- make sure fprintd service is configured and enabled
```
Comment (RoyalOughtness) [inline files/system/usr/etc/authselect/custom/secureblue/REQUIREMENTS:1]: what will happen if it isn't?

Comment (wandapeter) [reply files/system/usr/etc/authselect/custom/secureblue/REQUIREMENTS:1]: fingerprint auth should simply not work but it shouldn't breake anything

Comment (RoyalOughtness) [reply files/system/usr/etc/authselect/custom/secureblue/REQUIREMENTS:1]: cool

File: files/system/usr/etc/authselect/custom/secureblue/dconf-db [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# dconf-db - DConf Database Configuration
+#
+# This file is part of the secureblue Authselect profile. It specifies the
+# configuration for DConf, which is a configuration system used to store
+# application settings. The settings in this file are applied as part of
+# the profile's system authentication and security configurations.
+
+[org/gnome/login-screen]
+enable-smartcard-authentication=false
```
Comment (fiftydinar) [inline files/system/usr/etc/authselect/custom/secureblue/dconf-db:9]: It's better to use `gschema-overrides` module for this, while dconf lock should remain where it is.

**Comment by RoyalOughtness** [comment]
@34N0 you'll need to point this at staging so I can merge it

**Comment by wandapeter** [comment]
> @34N0 you'll need to point this at staging so I can merge it

sorry i thought changing base is something maintainers can do as well :smile:

**Comment by RoyalOughtness** [comment]
> > @34N0 you'll need to point this at staging so I can merge it
> 
> sorry i thought changing base is something maintainers can do as well 😄

oh, you're right it is :smile:

**Review by RoyalOughtness — APPROVED** [review]

**Pull request merged**